src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -67,7 +67,7 @@
     {% if addon.current_version %}
       <li>
         <strong>{{ _('Listed Version:') }}</strong>
-        <a href="{{ addon.current_version.get_url_path() }}">{{ addon.current_version.version }}</a>
+        <a href="{{ addon.versions_url }}">{{ addon.current_version.version }}</a>
```

Comment:
this is just a drive-by change, right?  (it's obviously not saving accessing `current_version`)

Reply:
The issue I was trying to solve is that `<Version>.get_url_path()` is accessing the `addon`, which may not have been loaded already, so I was hunting for all calls to `<Version>.get_url_path()` to figure out where the problematic ones were coming from. But that was before I ended up setting `addon` in arguments loaded by `ActivityLog`, and this one in particular should be safe, since `current_version` should be coming from the `Addon` transformer. I'll try removing it and see if that changes anything.

Reply:
perhaps out of scope for this patch, but `Version.get_url_path` should probably just call `Addon.get_url_path` anyway, rather than replicate the `reverse`

Reply:
reverted in f896d03c704fefc4abb1cf471fdf7955a2a3e4fe, it's not necessary.